src/components/Calculator.js

```diff
@@ -1,25 +1,34 @@
+/* eslint-disable react/state-in-constructor */
 /* eslint-disable react/prefer-stateless-function */
 /* eslint-disable react/destructuring-assignment */
 /* eslint-disable react/prop-types */
-import React from 'react';
```

Comment:
- [ ] Let's not disable linter rules. These are here to help us write clean piece of code.